src/mugs.js

```diff
@@ -1445,6 +1445,18 @@ define([
         }
     });
 
+    var MicroImage = util.extend(Audio, {
```

Comment:
Can this extend from Image instead of Audio?

Reply:
I believe that is happening because of the `spec` of the Image which would need to over written though that makes it quite dirty.

Extending Audio for Image is confusing to read, may be it should be a `BaseMedia` object that should be extended for all. Though I see that is what we have been doing elsewhere as well. So not something you should address in this PR.

Reply:
> may be it should be a `BaseMedia` object that should be extended for all

This is a nice idea, and I think would help make things more readable. But I agree, since there is already pre-existing classes extending from `Audio`, this change is better handled in a separate PR.